var/spack/repos/builtin/packages/elfutils/package.py

```diff
@@ -50,6 +53,14 @@ class Elfutils(AutotoolsPackage):
     # elfutils with gcc, and then link it to clang-built libraries.
     conflicts('%clang')
 
+    # Elfutils uses -Wall and we don't want to fail the build over a
+    # stray warning.
+    def patch(self):
+        files = glob.glob(os.path.join('*', 'Makefile.in'))
+        filter_file('-Werror', '', *files)
```

Comment:
I always get quiesy when I read such a comment. Do you have a concrete reason why you needed to disable it?

Reply:
@healther, in this case I'll merge the package, but if you like I can raise an issue to start a discussion on when stripping `-Werror` is appropriate. It's possible this should be optional (e.g. controlled with a variant)